app/pages/project/stats/charts.cjsx

```diff
@@ -124,7 +124,17 @@ Graph = React.createClass
         if data.index % numberBars
           data.element.attr({style: "display: none"})
     else if data.type == 'bar'
-      data.element.attr({style: "stroke-width: #{100 / length}%"})
+      data.element.attr(
+        style: "stroke-width: #{100 / length}%"
+        focusable: true
```

Comment:
Sounds like Firefox hasn't added support for `tabindex` on SVG elements. I'll have a look to see if I can find anything about SVG support in the various browsers.